rest-service/manager_rest/resource_manager.py

```diff
@@ -712,7 +712,7 @@ def execute_workflow(self,
             for component_dep_id in components_dep_ids:
                 self.execute_workflow(component_dep_id,
                                       workflow_id,
-                                      blueprint_id,
+                                      None,
```

Comment:
@tyacbovi how about fetch the deployment and pass the right blueprint id, no reason to do a half-solution?

Reply:
Well it does that :)
https://github.com/cloudify-cosmo/cloudify-manager/blob/master/rest-service/manager_rest/resource_manager.py#L635

Reply:
:+1: